daemon/logic/worklog.go

```diff
@@ -112,7 +129,21 @@ func (w *Worklog) Resolve(ctx context.Context, orgID *identity.ID,
 			State:   apitypes.ManualWorklogResult,
 			Message: "Please set a new value for the secret at " + item.Subject,
 		}, nil
+	case apitypes.MissingKeypairsWorklogType:
+		err = w.engine.GenerateKeypair(ctx, n, orgID)
+		if err != nil {
```

Comment:
This will end up hiding the actual underlying error - how do we get at that for support / provide information to our users (e.g. their session expired so they need to login)